gradle.properties

```diff
@@ -51,7 +51,7 @@ javaPoetVersion=1.13.0
 shadowPluginVersion=7.0.0
 
 # Test dependencies
-jmhCoreVersion=1.30
+jmhCoreVersion=1.33
 jmhPluginVersion=0.6.4
```

Comment:
the jmh plugin is now at version 0.6.6 (https://github.com/melix/jmh-gradle-plugin/releases)

Reply:
I tried 0.6.5 in the past and there was an issue where the artifacts weren't published correctly, and took a little bit to figure out the issue. hopefully this issue has been resolved.

Reply:
https://github.com/apple/servicetalk/pull/1790